main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/pkg/errors"
+
+	"github.com/getsentry/sentry-go"
+
+	"github.com/theartofeducation/go-template-repo/app"
+
+	"github.com/gorilla/mux"
+	"github.com/joho/godotenv"
+
+	"github.com/sirupsen/logrus"
```

Comment:
Similar to the question above ... really very minor, but my OCD would definitely appreciate consistency in the whitespace with the imports of these external dependencies.

Reply:
I'm not sure which tool is doing it 💩